conda/environment.yml

```diff
@@ -3,27 +3,164 @@
 
 # !!! If you are developing the website locally use the existing lock files
 # for your platform (`conda create --name hackweek --file conda-osx-64.lock`)
+
+# Modified from https://github.com/CryoInTheCloud/hub-image/blob/main/environment.yml for UW Hackweek 2024
+
 name: hackweek
+
 channels:
   - conda-forge
+  - nodefaults
+
 dependencies:
-  - ipyleaflet
-  - jupyter-book<2
-  - jupyter-resource-usage
-  - jupyterhub-singleuser
-  - jupyterlab
-  - jupytext
-  - pip
-  - python
+  - python~=3.11.0
+  # Required until https://github.com/jupyterhub/repo2docker/pull/1196 is merged
+  - jupyterhub-singleuser~=4.0.1
+  # nbgitpuller is very helpful when distributing user content
+  - nbgitpuller~=1.2.0
+  # Specify major version of jupyterlab to use. Manually bump it when you want to upgrade
+  # jupyterlab version
+  - jupyterlab>=4.0,<4.1
+  # This provides memory usage stats on JupyterLab and classic notebook
+  - jupyter-resource-usage~=1.0.1
+  # https://syncthing.net/ provides a dropbox-like interface for syncing directories across users,
+  # hubs and machines. Used along with `jupyter-syncthing-proxy`.
+  # - jupyter-syncthing-proxy==1.0.3
+  # - syncthing~=1.22.1
+  # gh-scoped-creds allows users to securely push to GitHub from their repo.
+  # https://blog.jupyter.org/securely-pushing-to-github-from-a-jupyterhub-3ee42dfdc54f has
+  # some more information
+  # - gh-scoped-creds==4.1
+  # More git tools from github-cli
+  - gh~=2.32.1
+
+  # VS Code support
+  # - jupyter-vscode-proxy~=0.5
+  # - code-server~=4.16.1
+
+  # Jupyter tools
+  - ipython~=8.20.0
+  - ipywidgets~=8.1.1
+  - jupyter-ai~=2.20
+  - jupyter-book~=1.0.0
+#  - jupyter-offlinenotebook~=0.2.2 # Outdated according to jupyterlab
+#  - jupyter-collaboration~=2.0.1
+  - jupyterlab-favorites~=3.2.1
+  - jupyterlab-geojson~=3.4.0
+  - jupyterlab-git~=0.50.0
+  - jupyterlab-h5web~=11.1.0
+  - jupyterlab-myst~=2.4.2
```

Comment:
Probably also don't need any of these extensions just to build the book

Reply:
I like having [jupyterlab-myst](https://github.com/jupyter-book/jupyterlab-myst) so that markdown is rendered the same way on website and in jupyterlab